include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl_radix_sort.h

```diff
@@ -673,7 +673,7 @@ struct __parallel_radix_sort_iteration
 
         // work-group size must be a power of 2 and not less than the number of states.
         // TODO: Check how to get rid of that restriction.
-        __count_wg_size = 
+        __count_wg_size =
```

Comment:
formatting only change

Reply:
Would you mind if I leave that change?

Reply:
I don't really mind.  I know others feel strongly to keep the git blame clean, but I am ok with fixing this formatting issue here.